src/handler/http/request/folders.rs

```diff
@@ -75,7 +76,8 @@ pub async fn create_folder(
 ) -> impl Responder {
     let org_id = path.into_inner();
     let folder = body.into_inner().into();
-    match folders::save_folder(&org_id, folder, false).await {
+    let folder_type = FolderType::Dashboards;
+    match folders::save_folder(&org_id, folder, folder_type, false).await {
```

Comment:
All endpoints in this route were originally written before we introduced different folder types, and for the time being that are only intended to work on dashboard folders.

In the future we will introduce new endpoints or update these existing ones to accept the folder type as a parameter.